common/djangoapps/third_party_auth/saml.py

```diff
@@ -41,10 +41,18 @@ def auth_url(self):
         if not self._config.enabled:
             log.error('SAML authentication is not enabled')
             raise Http404
+
         # TODO: remove this check once the fix is merged upstream:
         # https://github.com/omab/python-social-auth/pull/821
         if 'idp' not in self.strategy.request_data():
             raise AuthMissingParameter(self, 'idp')
+
+        from .models import SAMLProviderConfig
```

Comment:
@bdero I might be misusing the term here, but I was talking about putting this import at the top of the file, rather than inside a method. If I'm not mistaking importing something inside method is called local import, while it seems to me you have tried to advocate for relative import (the one that starts with `.` rather than top-level package).